From def1b3bbe8e01beee95720e2804df4509c7e72f0 Mon Sep 17 00:00:00 2001
From: Jason Liu <r64343@freescale.com>
Date: Tue, 10 Sep 2013 10:17:37 +0800
Subject: [PATCH 416/505] ENGR00278921 linux-test: mxc_i2c_test: exclude the
 build for IMX6Q and IMX6S

This test is not valuable on i.MX6Q and IMX6S now, we can use the common
i2c-tools to do the i2c test now.

Signed-off-by: Jason Liu <r64343@freescale.com>
---
 test/mxc_i2c_test/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_i2c_test/Makefile b/test/mxc_i2c_test/Makefile
index b524577..4424551 100644
--- a/test/mxc_i2c_test/Makefile
+++ b/test/mxc_i2c_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX35_3STACK IMX25_3STACK"
+EXCLUDE_LIST:="IMX35_3STACK IMX25_3STACK IMX6Q IMX6S"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = \
-- 
1.8.0

